app/components/CapitalProjectDetailPanel/CapitalProjectDetailPanel.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { CapitalProjectDetailPanel } from "./CapitalProjectDetailPanel";
+import {
+  CapitalProjectBudgeted,
+  createCapitalProjectBudgeted,
+  Agency,
+} from "~/gen";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("CapitalProjectDetailPanel", () => {
```

Comment:
Interesting, we group tests differently. I'm lazy and group all the "thing should be there" assertions together in a single test. It seems you more strictly follow the "one assertion per test". Respect. I suppose I'm more lax about grouping them because I feel the assertions are expressive enough that I can identify which assertion failed within a test. Anyway, neat.